packages/fast-development-site-react/src/components/shell/header.tsx

```diff
@@ -17,9 +17,10 @@ const style: ComponentStyles<IShellHeaderManagedClasses, IDevSiteDesignSystem> =
         color: (config: IDevSiteDesignSystem): string => {
             return config.backgroundColor;
         },
-        textAlign: "center",
-        padding: "12px",
-        minHeight: "24px"
+        textAlign: "left",
+        padding: "3px",
```

Comment:
Same here and all other instances - need to use `toPx()`

Reply:
Same as above

Reply:
`toPx()` will be addressed in another PR